typescript/infra/config/environments/mainnet3/validators.ts

```diff
@@ -896,5 +896,76 @@ export const validatorChainConfig = (
         'oortmainnet',
       ),
     },
+
+    immutablezkevm: {
+      interval: 5,
+      reorgPeriod: getReorgPeriod('immutablezkevm'),
+      validators: validatorsConfig(
+        {
+          [Contexts.Hyperlane]: ['0xa787c2952a4d22f776ee6e87e828e6f75de24330'],
```

Comment:
I assume onboarding other validators comes later?

Reply:
correct - our deployment process is split into two parts:
1. initial contracts + AW agents
2. enrolling validators + updating ISMs